main.py

```diff
@@ -0,0 +1,21 @@
+import asyncio
+
+from handling_presentations import extract_text_from_presentation
+from handling_api import api_request
+
+CONTENT = "Text below from presentation slide give me short explanation about this topic \n"
```

Comment:
I feel this should be in the GPT module.
it can probably be a part of the string itself, unless you want to allow modularity in the API responses?
Also, why do you need this if you have the "system" role in the message for the API request?